doc/source/operations/upgrading-openstack.rst

```diff
@@ -337,9 +285,69 @@ Ubuntu Noble migration
 ----------------------
 
 Ubuntu Jammy support has been removed from the 2025.1 release onwards. Hosts
-must be migrated to Ubuntu 24.04 before upgrading OpenStack services. The
-upgrade process is currently a work in progress.
-.. TODO: Add link to another page describing how to migrate
+must be migrated to Ubuntu 24.04 before upgrading OpenStack services.
+You can find the upgrade procedure from :ref:`upgrading-to-ubuntu-noble`
+documentation.
+
+RabbitMQ 4.1
+------------
+
+.. warning::
+
+   StackHPC Kayobe Config sets RabbitMQ 4.1 as the default for the Epoxy release.
+   Existing transient queues must be migrated to durable queues with Queue Manager
+   before upgrading to RabbitMQ 4.1.
```

Comment:
Apologies, I've actually changed my mind.

I think if we change the heading, and clearly state the order of operations, it will make it much clearer for the reader.

```suggestion

RabbitMQ Prerequisites
----------------------

.. warning::

   StackHPC Kayobe Config sets RabbitMQ 4.1 as the default for the Epoxy release.
   Existing transient queues must be migrated to durable queues with Queue Manager
   before upgrading to RabbitMQ 4.1.
   
   This means that queue migration and the RabbitMQ 4.1 upgrade must be completed
   before upgrading to Epoxy.
```

Reply:
It actually renders fine as is:
<img width="1125" height="365" alt="image" src="https://github.com/user-attachments/assets/d8aed5be-c60a-4a5d-a15c-819a3691404d" />
I'd prefer to keep it as two separate paragraphs if possible